api/app/lib/statistics.rb

```diff
@@ -0,0 +1,104 @@
+require "google/apis/analytics_v3"
+require "google/api_client/auth/key_utils"
+
+class Statistics
+  include ActiveModel::Validations
+  include ActiveModel::Conversion
+  extend ActiveModel::Naming
+
+  def id
+    "statistics-set"
+  end
+
+  def initialize(_range = nil)
+    @client = Google::Apis::AnalyticsV3::AnalyticsService.new
+
+    @client.authorization = Signet::OAuth2::Client.new(
+      token_credential_uri:  "https://accounts.google.com/o/oauth2/token",
+      audience:              "https://accounts.google.com/o/oauth2/token",
+      scope:                 "https://www.googleapis.com/auth/analytics.readonly",
+      issuer:                 ENV["GOOGLE_SERVICE_CLIENT_EMAIL"],
+      signing_key:            OpenSSL::PKey::RSA.new(ENV["GOOGLE_SERVICE_PRIVATE_KEY"])
```

Comment:
Confirm that these env vars exist in .env.sample. If they don't, they should be added.

Also, what happens if these variables aren't set, or if the authentication fails? Does this class handle it gracefully?